tests/EventLoopSpec.hs

```diff
@@ -837,7 +837,7 @@ eventLoopSpec = parallel $ do
         , "* c0"
         ]
 
-    it "does not apply fixup commits if the commit to fix is not on the branch" $ do
+    it "do not merge if there exist fixup commits that do not belong to any other commits" $ do
       history <- withTestEnv $ \ shas runLoop git -> do
```

Comment:
Would it be possible to also check that it leaves the right comments? Not just that it refuses the merge.

Reply:
There is no straightforward way to add such checks to tests. Created a new issue #92.